syntax.in

```diff
@@ -1,31 +0,0 @@
-<cmpstmt> ::= <stmt> | <stmt> ; <cmpstmt>
-<stmt> ::= <assignStmt> | <structStmt> | <funcDeclStmt>
-<assignStmt> ::= <identifier>=<expression>
-// no IO statements, they are instead functions part of the standard library
```

Comment:
debatable, I had in mind something like c with printf, I think it's more flexible to make it a function, since I would want to abstract over this "statement" anyway, for example specify how much to read or from what input, would probably end up with functions anyway